redaxo/src/core/lib/form/form_base.php

```diff
@@ -935,15 +935,15 @@ protected function getElement($fieldsetName, $elementName)
 
     /**
      * @param string $fieldsetName
-     * @param string $elementName
+     * @param string $normalizedElementName
      *
      * @return rex_form_element|null
      */
-    private function _getElement($fieldsetName, $elementName)
+    private function _getElement($fieldsetName, $normalizedElementName)
```

Comment:
Um den unter§chied zu getElement() hervorzuheben

Reply:
Wenn ich mir den Code so anschaue, frage ich mich, wozu es diese Methode überhaupt gibt.
Die wird ja scheinbar nur in `getElement` aufgerufen.
Und da spart es weder Code, noch tiefe Einrückungen oder so.
Von daher könnte der Code eigentlich auch direkt mit in `getElement` rein, oder?

Reply:
Guter Punkt